presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -168,58 +168,73 @@ Configuration Properties
-                                                   aggregations or joins.
-
-``iceberg.enable-merge-on-read-mode``              Enable reading base tables that use merge-on-read for          ``true``
-                                                   updates.
-
-``iceberg.delete-as-join-rewrite-enabled``         When enabled, equality delete row filtering is applied        ``true``
-                                                   as a join with the data of the equality delete files.
-================================================== ============================================================= ============
+======================================================= ============================================================= ============
+Property Name                                           Description                                                   Default
+======================================================= ============================================================= ============
+``hive.metastore.uri``                                  The URI(s) of the Hive metastore to connect to using the
+                                                        Thrift protocol. If multiple URIs are provided, the first
+                                                        URI is used by default, and the rest of the URIs are
+                                                        fallback metastores.
+                                                        Example: ``thrift://192.0.2.3:9083`` or
+                                                        ``thrift://192.0.2.3:9083,thrift://192.0.2.4:9083``
+                                                        This property is required if the
+                                                        ``iceberg.catalog.type`` is ``hive``. Otherwise, it will
+                                                        be ignored.
+
+``iceberg.file-format``                                 The storage file format for Iceberg tables. The available     ``ORC``
+                                                        values are ``PARQUET`` and ``ORC``.
+
+``iceberg.compression-codec``                           The compression codec to use when writing files. The          ``GZIP``
+                                                        available values are ``NONE``, ``SNAPPY``, ``GZIP``,
+                                                        ``LZ4``, and ``ZSTD``.
+
+``iceberg.catalog.type``                                The catalog type for Iceberg tables. The available values     ``hive``
+                                                        are ``hive``, ``hadoop``, and ``nessie`` corresponding to
+                                                        the catalogs in Iceberg.
+
+``iceberg.catalog.warehouse``                           The catalog warehouse root path for Iceberg tables.
+                                                        ``Example: hdfs://nn:8020/warehouse/path``
```

Comment:
```suggestion
                                                        
                                                        Example: ``hdfs://nn:8020/warehouse/path``
```
Suggest add a line to left-justify the Example (as it seems you may have meant to), and reformat to remove "Example: " from the code formatting.